SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -238,13 +252,20 @@ LAYEREDLBSTRATEGY_TEST(
 	ADD_LATENT_AUTOMATION_COMMAND(FWaitForWorld(Data));
 	ADD_LATENT_AUTOMATION_COMMAND(FCreateStrategy(Data, MultiWorkerSettings, {}));
 	ADD_LATENT_AUTOMATION_COMMAND(FCheckMinimumWorkers(Data, this, 10));
+	ADD_LATENT_AUTOMATION_COMMAND(FCleanup(Data));
+
+	Settings->bAutoStartLocalDeployment = OldBAutoStartLocalDeployment;
```

Comment:
Beware that this will actually run right after AutomationOpenMap. Which is probably fine, since the map would have been opened with the correct setting by then, but it's worth keeping in mind, and may be better to reorder these things, so the order of operations is clearer.